.circleci/config.yml

```diff
@@ -65,9 +63,7 @@ commands:
       - run:
           name: 'Configure Hosts'
           command: cat ./build-system/test-configs/hosts | sudo tee -a /etc/hosts
-      - run:
-          name: 'Install Dependencies'
-          command: ./.circleci/install_dependencies.sh
+      - node/install-packages
```

Comment:
While there is definitely a speed advantage to using this orb, one of the visible downsides is that what used to be one step now occupies five steps, causing the job logs to be more noisy and less readable. This was why the original implementation uses a bash script that contains just what we need and nothing else.

**Before:**

![image](https://user-images.githubusercontent.com/26553114/125362160-6d866800-e33c-11eb-9323-b2b7deafbaf0.png)

**After:**

![image](https://user-images.githubusercontent.com/26553114/125362199-7d05b100-e33c-11eb-86ca-d20f55a585d9.png)

Curious: Would it be possible to do something similar here and retain the speed up while not growing the number of steps by extracting just the necessary steps from the orb?

Reply:
I understand the concern, but I think the payoff is worth the slight inconvenience. Honestly it would be great if CircleCI had an option to group steps together so that we can place all the setup steps together - maybe worth sending an FR?

Reply:
I added commit 3d9b1fd258302223f5fffcc93a832affb22badf6, which merges several steps together to reduce overall steps count. wdyt?

Reply:
I like those other changes (thank you!), but still think we should attempt not to expand node package installation from 1 step to 5 steps. It's more troublesome to write this PR, but makes the job logs significantly easier to read, especially for new developers.

If speeding up package installation is independent from switching to docker containers, can we move this part into its own PR? WDYT?

Reply:
Would you mind if I submit this PR with this change and work on reducing the number of steps in a followup PR?

Reply:
No objections at all. Thanks for humoring me 😃